packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -388,6 +388,13 @@ contract DeployImplementations is Script {
             info: OPStackManager.Implementation(address(_dio.l1StandardBridgeImpl()), L1StandardBridge.initialize.selector)
         });
 
+        setters[6] = OPStackManager.ImplementationSetter({
+            name: "DisputeGameFactory",
+            info: OPStackManager.Implementation(
+                address(_dio.disputeGameFactoryImpl()), DisputeGameFactory.initialize.selector
+            )
+        });
```

Comment:
the actual impl was already deployed in this script, just wasn't being provided to the OPSM's initializer.